src/lib/OpenApi.ts

```
@@ -242,6 +242,32 @@ export function OpenApi({
+  }
+
+  function getOperations() {
+    const paths = getSpec().paths as OpenAPIV3.PathsObject
+
+    if (!paths) {
+      return []
+    }
+
+    return Object.entries(paths).flatMap(([_, methods]) => {
+      return httpVerbs
+        .filter(verb => methods && methods[verb])
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)